sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIClientOptions.cs

```diff
@@ -1,17 +1,43 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.ClientModel.Primitives;
 using OpenAI;
 
 namespace Azure.AI.OpenAI;
 
 /// <summary>
 /// Defines the scenario-independent, client-level options for the Azure-specific OpenAI client.
 /// </summary>
-public partial class AzureOpenAIClientOptions : OpenAIClientOptions
+public partial class AzureOpenAIClientOptions : ClientPipelineOptions
 {
-    internal string Version => _version;
-    private readonly string _version;
+    internal string Version { get; }
+
+    /// <summary>
+    /// Specifies custom authorization scopes to use when authenticating a client with a supplied <see cref="Azure.Core.TokenCredential"/>.
```

Comment:
2 things about this new public API surface:
1. Mention this new feature in the changelog.
2. Document why and how to use this feature in the README. The [Authenticate the client](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/openai/Azure.AI.OpenAI#authenticate-the-client) section might be the best place for it.

Reply:
Updated!